### PR TITLE
TravisEz13_181204_214715.808

### PR DESCRIPTION
**Pull request by TravisEz13**
**Type:** Missing

**Summary:**
Set the license info where it was missing

**Details:**
Set the license info where it was missing on many packages

**Resolution:**
Set the license info where it was missing

**Affected definitions**:
- Markdig.Signed 0.15.4,- Microsoft.ApplicationInsights 2.8.1,- Microsoft.Win32.Registry.AccessControl 4.5.0,- Microsoft.Win32.SystemEvents 4.5.0,- Microsoft.Windows.Compatibility 2.0.1,- NJsonSchema 9.12.2,- System.CodeDom 4.5.0,- System.ComponentModel.Composition 4.5.0,- System.Configuration.ConfigurationManager 4.5.0,- System.Data.DataSetExtensions 4.5.0,- System.Data.Odbc 4.5.0,- System.Diagnostics.EventLog 4.5.0,- System.Diagnostics.PerformanceCounter 4.5.0,- System.DirectoryServices 4.5.0,- System.DirectoryServices.AccountManagement 4.5.0,- System.DirectoryServices.Protocols 4.5.0,- System.Drawing.Common 4.5.0,- System.IO.FileSystem.AccessControl 4.5.0,- System.IO.Packaging 4.5.0,- System.IO.Pipes.AccessControl 4.5.0,- System.IO.Ports 4.5.0,- System.Management 4.5.0,- System.Net.Http.WinHttpHandler 4.5.1,- System.Private.ServiceModel 4.5.3,- System.Reflection.DispatchProxy 4.5.0,- System.Runtime.Caching 4.5.0,- System.Runtime.CompilerServices.Unsafe 4.5.2,- System.Security.Cryptography.Pkcs 4.5.1,- System.Security.Cryptography.ProtectedData 4.5.0,- System.Security.Principal.Windows 4.5.1,- System.ServiceModel.Duplex 4.5.3,- System.ServiceModel.Http 4.5.3,- System.ServiceModel.NetTcp 4.5.3,- System.ServiceModel.Primitives 4.5.3,- System.ServiceModel.Security 4.5.3,- System.ServiceModel.Syndication 4.5.0,- System.ServiceProcess.ServiceController 4.5.0,- System.Threading.AccessControl 4.5.0

### CODE REVIEW

**Comment by TravisEz13** [comment]
Created to fix merge conflicts with #847

**Comment by TravisEz13** [comment]
You can review the change introduced to the full definition at [ClearlyDefined](https://clearlydefined.io/curations/848).

**Comment by TravisEz13** [comment]
cc @capfei

File: curations/nuget/nuget/-/Markdig.Signed.yaml [hunk 1]
```diff
@@ -9,4 +9,6 @@ revisions:
   0.15.2:
     licensed:
       declared: BSD-2-Clause
-      
\ No newline at end of file
+  0.15.4:
+    licensed:
+      declared: BSD-2-Clause
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/Markdig.Signed.yaml]: verified license at https://github.com/lunet-io/markdig/blob/master/license.txt

File: curations/nuget/nuget/-/Microsoft.ApplicationInsights.yaml [hunk 2]
```diff
@@ -23,7 +23,10 @@ revisions:
...
   2.7.1:
     licensed:
-      declared: MIT    
+      declared: MIT
   2.7.2:
     licensed:
       declared: MIT
+  2.8.1:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/Microsoft.ApplicationInsights.yaml]: Verified via License Info link at https://www.nuget.org/packages/Microsoft.ApplicationInsights

File: curations/nuget/nuget/-/NJsonSchema.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: NJsonSchema
+  provider: nuget
+  type: nuget
+revisions:
+  9.12.2:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/NJsonSchema.yaml]: verified at https://github.com/rsuter/NJsonSchema/blob/master/LICENSE.md

File: curations/nuget/nuget/-/System.CodeDom.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.CodeDom
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.CodeDom.yaml]: https://www.nuget.org/packages/System.CodeDom links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.ComponentModel.Composition.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.ComponentModel.Composition
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.ComponentModel.Composition.yaml]: https://www.nuget.org/packages/System.ComponentModel.Composition links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.Configuration.ConfigurationManager.yaml [hunk 6]
```diff
@@ -6,3 +6,6 @@ revisions:
   4.4.0:
     licensed:
       declared: MIT
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.Configuration.ConfigurationManager.yaml]: https://www.nuget.org/packages/System.Data.DataSetExtensions links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.Data.DataSetExtensions.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.Data.DataSetExtensions
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.Data.DataSetExtensions.yaml]: https://www.nuget.org/packages/System.Data.DataSetExtensions links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.Data.Odbc.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.Data.Odbc
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.Data.Odbc.yaml]: https://www.nuget.org/packages/System.Data.Odbc links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.Diagnostics.EventLog.yaml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.Diagnostics.EventLog
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```
Comment (TravisEz13) [inline curations/nuget/nuget/-/System.Diagnostics.EventLog.yaml]: https://www.nuget.org/packages/System.Diagnostics.EventLog links to 
https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT

File: curations/nuget/nuget/-/System.Data.DataSetExtensions.yaml (hunk 7 above)
Comment (TravisEz13) [reply curations/nuget/nuget/-/System.Data.DataSetExtensions.yaml]: I followed a process like this for the rest of this items.  Please ask me if you have questions

**Comment by capfei** [comment]
Closing since #847 has been merged

**Pull request closed without merging**